### ISSUE TITLE
1.16.1: When use Bungee_Chat mode,DiscordSRV plugin can't can't receive any message

### ISSUE THREAD

**Issue by MiunaPA** [kangarko/ChatControl-Pro#1744]
(My English is not very good, please forgive me if I have grammatical problems)

Server : 1.16.1 paper-26
Plugin : ChatControl-9.0.2-BETA

In the past, when I was using an older version(1.15.2 & ChatControl8.6.6), I used DiscordSRV to transfer the game chat to Discord(Also enable Bungee_Chat).

It looks like this (in old version):
![](https://i.imgur.com/cdurBan.png)
Messages can be sent to Discord using Discord SRV's webhook function, also can sent to other servers using same chat format.

But when I using new version, when player switch to bungee_chat mode, DiscordSRV can't receive any message
It looks like this (in new version):
![](https://i.imgur.com/IpI6ZGY.png)
In bungee_chat mode, only ChatControl's discord features can use,and DiscordSRV's webhook function has no effect.

The message I want to display in Discord is to use DiscordSRV's webhook function, but at the same time it can be enabled in bungee_chat mode.

I can do this in the old version, but how does this work on the new version?

This is my setting.yml and ChatControl folder
https://pastebin.com/2bs17NSB
[ChatControl.zip](https://github.com/kangarko/ChatControl-Pro/files/4850400/ChatControl.zip)

**Comment by lackoSK** [comment]
Hello,

Can your set debug to [discord]  (settings.yml)

Sincerely,
Ladislav from MineAcademy.org

**Comment by MiunaPA** [comment]
I found that in the old version (8.6.6), when the player turned on bungee chat mode, two events would occur (the canceled state is true once and false once), while in game chat mode, only There is an uncancelled event.

But in the new version (9.0.2-beta), when bungee chat mode is turned on, there is only one canceled chat event (instead of two events, one true and one false), while in game chat mode, there is only one uncanceled event.

At present, I set DiscordSRV's config to `RespectChatPlugins: false`, which allows DiscordSRV to send webhook messages normally.I am not sure whether the past two events are a kind of bug.

Two versions of debug messages are attached here.
![](https://i.imgur.com/HxTSA6S.png)
![](https://i.imgur.com/PLyUaUL.png)
(At 1.15.2 ,DiscordSRV's config set `RespectChatPlugins: true` , 1.16.1 set `RespectChatPlugins: false`)

But now the new version (9.0.2-beta) has been launched, maybe the problem has been solved.

Sorry I may have asked a very stupid question, now I can use the webhook function as before, thank you :)
